### PR TITLE
Fetch Contributions python script

### LINKED ISSUE #1171: Python: Fetch all the Pull Requests created by a User on a GitHub Organization and store it in a Markdown Table
# Description

Create a Script to fetch all the Pull Requests created by a User on a GitHub Organization and store it in a Markdown Table.

The script should take the username and the organization name as an argument and result in a markdown file with a table with the following content: 

- Title of PR
- Link of PR
- Status (Merged/Closed)

## Language

- [X] Python

### PR DESCRIPTION
**Pull request by kadatatlukishore**
# Description
#Fixes  #1171 
Created a Script to fetch all the Pull Requests created by a User on a GitHub Organization and store it in a Markdown Table.

The script should take the username and the organization name as an argument and result in a markdown file with a table with the following content:

- Title of PR
- Link of PR
- Status (Merged/Closed)
# Type of change
  - [x] New feature (non-breaking change which adds functionality)
# Checklist:
 - [x] My code follows the style guidelines(Clean Code) of this project
 - [x] I have performed a self-review of my own code
  - [x] I have commented my code, particularly in hard-to-understand areas
 - [x]  I have created a helpful and easy to understand README.md
  - [x] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings
  - [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Fetch_Contributions/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Fetch contributions
+
+- Script can be executed as a CLI.
+- Takes arguments username and organization.
+- Returns the pull requests made by user to the organization into a Markdown file. 	
+	- Markdown file consists a table having - "Title to PR" , "Link of PR" , "Status(Merged/Closed/Open)"
+
+## Example Markdown File:
```
Comment (vybhav72954) [inline Python/Fetch_Contributions/README.md:8]: You have already uploaded the md file, no need to repeat the same in the README> Just provide a link to the file here?

File: Python/Fetch_Contributions/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+python3 -m venv env
+source env/bin/activate
+pip install -r requirements.txt
+python3 fetch_contributions.py --username <Username> --organization <Organization>
+
+Example: 
+>>> python3 fetch_contributions.py --username <HarshCasper> --organization moja-global
+ 
+```
+We will get the resuts in a file like **above** table.
```
Comment (vybhav72954) [inline Python/Fetch_Contributions/README.md:43]: Consider adding an author section as well?

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Everything looks good, although the README could be a bit better maybe? Take inspiration from some other REAMEs in the projects? ALthough it looks good right now as well, but in case you can add some more info or something, would be great.

**Comment by vybhav72954** [comment]
Also please use `#Fixes <issue number` in the PR body

**Comment by kadatatlukishore** [comment]
Hey @vybhav72954  , I am a newbie. Can I create another pull request need to do some changes or Can I change in repository?

**Comment by vybhav72954** [comment]
> Hey @vybhav72954  , I am a newbie. Can I create another pull request need to do some changes or Can I change in repository?

Seems like you figured it out. Awesome

**Comment by kadatatlukishore** [comment]
@vybhav72954 Thanks. Yes, required changes are made.

**Comment by kadatatlukishore** [comment]
Hey @vybhav72954 , I made all the necessary changes. Only one security issue is there for `from lxml import html` line. I've used to parse html using xpath. So, most of the code has been written using it.

**Comment by HarshCasper** [comment]
> Hey @vybhav72954 , I made all the necessary changes. Only one security issue is there for `from lxml import html` line. I've used to parse html using xpath. So, most of the code has been written using it.

Don't worry about security issues.

**Comment by kadatatlukishore** [comment]
> > Hey @vybhav72954 , I made all the necessary changes. Only one security issue is there for `from lxml import html` line. I've used to parse html using xpath. So, most of the code has been written using it.
> 
> Don't worry about security issues.

Thanks @HarshCasper

File: Python/Fetch_Contributions/markdown_file.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+|    | Title to PR                                               | Link of PR                                                            | Status(Merged/Closed/Open)   |
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/markdown_file.md:1]: Remove this file. It contains my contributions :sweat_smile:

File: Python/Fetch_Contributions/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Fetch contributions
+
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:2]: Can you add a simple introduction in a few lines?

File: Python/Fetch_Contributions/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- Script can be executed as a CLI.
+- Takes arguments username and organization.
+- Returns the pull requests made by user to the organization into a Markdown file. 	
+	- Markdown file consists a table having - "Title to PR" , "Link of PR" , "Status(Merged/Closed/Open)"
+
+## Example Markdown File:
+
+|    | Title to PR                                               | Link of PR                                                            | Status(Merged/Closed/Open)   |
+|---:|:----------------------------------------------------------|:----------------------------------------------------------------------|:-----------------------------|
+|  0 | Format Code using Black                                   | https://github.com/moja-global/FLINT.Cloud/pull/22                    | Merged                       |
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:12]: Remove this

File: Python/Fetch_Contributions/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+See [Here](https://github.com/kadatatlukishore/Rotten-Scripts/blob/fetch-pullrequests/Python/Fetch_Contributions/markdown_file.md)
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:14]: Remove this

File: Python/Fetch_Contributions/README.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:16]: Remove unnecessary lines

File: Python/Fetch_Contributions/README.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## Setup Instructions:
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:18]: Add a line break

File: Python/Fetch_Contributions/README.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+The Code is written in Python 3.9. If you don't have Python installed you can find it [here](https://www.python.org/downloads/).
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:19]: Remove this line

File: Python/Fetch_Contributions/README.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+python3 fetch_contributions.py --username <Username> --organization <Organization>
+
+Example: 
+>>> python3 fetch_contributions.py --username HarshCasper --organization moja-global
+ 
+```
+We will get the resuts in a file like **above** table.
+
+# AUTHOR 
+**K Kishore**
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:35]: Add a hyperlink to your LinkedIn/GitHub profile :wink:

File: Python/Fetch_Contributions/README.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+python3 -m venv env
+source env/bin/activate
+pip install -r requirements.txt
+python3 fetch_contributions.py --username <Username> --organization <Organization>
+
+Example: 
+>>> python3 fetch_contributions.py --username HarshCasper --organization moja-global
+ 
+```
+We will get the resuts in a file like **above** table.
```
Comment (HarshCasper) [inline Python/Fetch_Contributions/README.md:32]: Remove this line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by kadatatlukishore** [comment]
Thanks for your inputs on this @HarshCasper . Made the necessary changes, please confirm. Thanks !

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**